singleplatform.php

```diff
@@ -105,15 +193,176 @@ function singlePlatformShortcode() {
         'sp-section-one'
     );
 
+    /* Background Color Fields */
+    add_settings_field(
+        'sp-primary-background-color',
+        'Primary color',
+        'singleplatformOptionPrimaryBgColor',
+        'sp-plugin',
+        'sp-background-colors-section'
+    );
+
+    add_settings_field(
+        'sp-secondary-background-color',
+        'Secondary color',
+        'singleplatformOptionSecondaryBgColor',
+        'sp-plugin',
+        'sp-background-colors-section'
+    );
+
+    add_settings_field(
+        'sp-tertiary-background-color',
+        'Tertiary color',
+        'singleplatformOptionTertiaryBgColor',
+        'sp-plugin',
+        'sp-background-colors-section'
+    );
+
+    /* Font Fields */
+    add_settings_field(
+        'sp-font-family',
+        'Font Family',
+        'singlePlatformOptionFontFamily',
+        'sp-plugin',
+        'sp-font-section'
+    );
+
+    add_settings_field(
+        'sp-base-font-size',
+        'Base font size',
+        'singleplatformOptionFontSize',
+        'sp-plugin',
+        'sp-font-section'
+    );
+
+    add_settings_field(
+        'sp-item-casing',
+        'Item casing',
+        'singleplatformOptionItemCasing',
+        'sp-plugin',
+        'sp-font-section'
+    );
+
+    /* Font Color Fields */
+    add_settings_field(
+        'sp-primary-font-color',
+        'Primary Font Color',
+        'singlePlatformOptionPrimaryFontColor',
+        'sp-plugin',
+        'sp-font-colors-section'
+    );
+
+    add_settings_field(
+        'sp-secondary-font-color',
+        'Secondary Font Color',
+        'singlePlatformOptionSecondaryFontColor',
+        'sp-plugin',
+        'sp-font-colors-section'
+    );
+
+    add_settings_field(
+        'sp-tertiary-font-color',
+        'Tertiary Font Color',
+        'singlePlatformOptionTertiaryFontColor',
+        'sp-plugin',
+        'sp-font-colors-section'
+    );
+
+    /* Display Fields */
+    add_settings_field(
+        'sp-display-announcements',
+        'Announcements',
+        'singleplatformOptionDisplayAnnouncements',
+        'sp-plugin',
+        'sp-display-section'
+    );
+
+    add_settings_field(
+        'sp-hide-display-option-photos',
+        'Photos tab',
+        'singlePlatformOptionHidePhotos',
+        'sp-plugin',
+        'sp-display-section'
+    );
+
+    add_settings_field(
+        'sp-display-dollar-sign',
+        'Dollar sign',
```

Comment:
I feel like maybe we should replace all instances of `dollar-sign` with `currency-sign` maybe.

Since we currently support at least the Euro currency sign

Just tested with this SPV2ID `basil-berlin` and it's the Euro.

What do you think?